### PR TITLE
Talal bakkour/issue42

### PR DESCRIPTION
**Pull request by talal-bakkour**

### CODE REVIEW

File: src/routes/dish.js [hunk 1]
```diff
@@ -112,9 +112,9 @@ router.put('/:id', dishController.updateDish)
  *           type: string
  *     responses:
  *       200:
- *         description: Dish successfully deleted
+ *         description: Dishe successfully deleted
```
Comment (Eng-NUREDDIN) [inline src/routes/dish.js:115]: it must be order successfully deleted not Dish

File: src/routes/dish.js [hunk 2]
```diff
@@ -112,9 +112,9 @@ router.put('/:id', dishController.updateDish)
...
  *       404:
- *         description: Dish not found
+ *         description: Dishe not found
```
Comment (Eng-NUREDDIN) [inline src/routes/dish.js:117]: same here Order not Dish

File: src/routes/order.js [hunk 3]
```diff
@@ -1,14 +1,129 @@
...
+ */
+
+ router.put('/:id', orderController.updateOrder);
+
+/**
+ * @swagger
+ * /api/order/{id}:
+ *   delete:
+ *     tags: [Order]
+ *     summary: Delete a dish by ID
```
Comment (Eng-NUREDDIN) [inline src/routes/order.js:109]: same here Order not Dish

File: src/routes/order.js [hunk 4]
```diff
@@ -1,14 +1,129 @@
...
+ * /api//Order/:
+ *   post:
+ *     tags: [Order]
+ *     summary: Add a new Order
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/Order_Add'
```
Comment (radmanlo) [inline src/routes/order.js:39]: Order_Add should be Schema in swagger.js file. I cannot see this schema in theswagger file.

File: src/routes/order.js [hunk 5]
```diff
@@ -1,14 +1,129 @@
...
+ *         name: id
+ *         required: true
+ *         schema:
+ *           type: string
+ *     requestBody:
+ *       required: true
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/Order_Update'
```
Comment (radmanlo) [inline src/routes/order.js:90]: Order_Update also be in swagger.js in schema part

**Review by radmanlo — CHANGES_REQUESTED** [review]

**Review by radmanlo — APPROVED** [review]

**Review by afrakucukaydin — APPROVED** [review]

**Pull request merged**